vendor/assets/stylesheets/dvl/core/typography.scss

```diff
@@ -6,18 +6,19 @@
    http://modularscale.com/scale/?px1=12&px2=20&ra1=1.333&ra2=0
 \* ================================================================== */
 
-html { font-size: 75%; } // 12px
-@media only screen and (min-width: 320px) { html { font-size: $maxFontSize * 0.8125; } } // 13px
```

Comment:
I'm pretty sure this is what's causing bad layouts in small viewports: https://github.com/dobtco/screendoor-v2/pull/2153#issuecomment-117712592